doc/modules/reference.rst

```diff
@@ -229,20 +229,20 @@ the :ref:`user guide <user_guide>` for more information and usage examples.
    load_confounds
    load_confounds_strategy
 
-.. _io_ref:
+.. _maskers_ref:
 
-:mod:`nilearn.input_data`: Loading and Processing Files Easily
-==============================================================
+:mod:`nilearn.maskers`: Loading and Processing fMRI data Easily
```

Comment:
```suggestion
:mod:`nilearn.maskers`: Loading and Processing fMRI Data Easily
```

I don't think I'd commit to fMRI data specifically though. Maybe "Neuroimaging" or "MR"?
However, if not, then data just needs to be capitalized since it's title case.

Reply:
maybe something like "extracting [data,signals,time series?] from (the relevant regions of) brain images"? this module isn't really about loading (from files) and "processing" is very generic

Reply:
I agree. I replaced by "Extracting Signals from Brain Images" in 2b94a62